_posts/2020-03-12-Steering-Group-executive-summary,-action-items,-and-minutes.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Steering Group executive summary, action items, and minutes
+author: Franklin Feingold
+display: true
+---
+
+# Steering Group executive summary, action items, and minutes
+
+Date: Thursday February 13, 2020
+Attended: Franklin Feingold, Melanie Ganz, Guiomar Niso, Robert Oostenveld, Kirstie Whitaker
+
+### Executive Summary
+
+We discussed the channels through which to share updates to the BIDS specification.
+An issue was opened to start crowdsourcing and collecting this information.
+We will be using this to share the release of v1.2.2 and to request the final rounds of feedback on the Genetic information extension.
+
+We discussed generating an example slide deck that  community members can  use (or pick from) when they want to present BIDS.
+
+We discussed considerations for when to open a repository under the BIDS standard organization on GitHub.
+There should be multiple people committed to the repository (otherwise it is better of under the personal GitHub account), the maintainers should be clearly identifiable (e.g. as a CODEOWNERS file at top level), there should be a code of conduct in the repo that is consistent with that of the BIDS specification, and (as a consequence of the code of conduct) the repository should welcome contributions from others.
+
+We want to host another community forum (but not too frequent) and potentially a starter kit/demo in the Open Science Room at OHBM.
+
+Regarding issue [#407](https://github.com/bids-standard/bids-specification/pull/407), we consider Zenodo to be a suitable  the location to store historical specification pdfs, also for future versions.
```

Comment:
instead of "regarding issue XXX"

better use "regarding the discussion on where historical pdfs should be hosted (see issue XXX) ..."